README.md

```diff
@@ -3,21 +3,176 @@
+| ----- | ------- | -------- | ------ | ------------ | ------------ |
+| Alice | Deposit | 1000 DAI | 1000   | 1000 DAI     | 1000         |
+| Bob   | Deposit | 1000 DAI | 1000   | 2000 DAI     | 2000         |
+| Vault | Lend    | 1800 DAI | 0      | 200 DAI      | 2000         |
+
+While the bet is active, Alice can redeem her 1000 shares if she chooses so, but will but the exchange rate per share will be significantly lower than 1 DAI per share. Her share is represented by the following equation:
+
+```text
+share = (totalAssets * shares) / totalShares
+```
+
+Eg:
+
+```text
+share = (200 * 1000) / 2000 = 100 DAI
+```
+
+Upon redeeming the shares will be burnt and Alice share balance will be 0. The Vault will have 1100 DAI in total assets and 1000 shares.
+
+4. Alice redeems 1000 shares for 100 DAI. Those shares are burnt.
+
+| User  | Action  | Amount   | Shares | Total Assets | Total Shares |
+| ----- | ------- | -------- | ------ | ------------ | ------------ |
+| Alice | Deposit | 1000 DAI | 1000   | 1000 DAI     | 1000         |
+| Bob   | Deposit | 1000 DAI | 1000   | 2000 DAI     | 2000         |
+| Vault | Lend    | 1800 DAI | 0      | 200 DAI      | 2000         |
+| Alice | Redeem  | 100 DAI  | (1000) | 100 DAI      | 1000         |
+
+The bet is now settled and the Vault has an exposure of 0 DAI and has made 1800 DAI on the losing bet. The total assets in the Vault are now 100 DAI. This would also hold true if the bet was a winning bet, and the market pays out the winning proposition.  The performance would be low as the bettor has one assets from the market.
+
+| User   | Action  | Amount     | Shares | Total Assets | Total Shares |
+| ------ | ------- | ---------- | ------ | ------------ | ------------ |
+| Alice  | Deposit | 1000 DAI   | 1000   | 1000 DAI     | 1000         |
+| Bob    | Deposit | 1000 DAI   | 1000   | 2000 DAI     | 2000         |
+| Vault  | Lend    | 1800 DAI   | 0      | 200 DAI      | 2000         |
+| Carol  | Back    | 1800 DAI   | 0      | 0 DAI        | 0            |
+| Market | Settle  | (3600) DAI | 0      | 200 DAI      | 2000         |
+| Carol  | Settle  | 3600 DAI   | 0      | 3600 DAI     | 0            |
+
+5. Vault settles the bet and has now 3700 DAI in total assets, while the total shares are still 1000 (HL-DAI).
+
+| User   | Action  | Amount   | Shares | Total Assets | Total Shares |
+| ------ | ------- | -------- | ------ | ------------ | ------------ |
+| Alice  | Deposit | 1000 DAI | 1000   | 1000 DAI     | 1000         |
+| Bob    | Deposit | 1000 DAI | 1000   | 2000 DAI     | 2000         |
+| Vault  | Lend    | 1800 DAI | 0      | 200 DAI      | 2000         |
+| Alice  | Redeem  | 100 DAI  | (1000) | 100 DAI      | 1000         |
+| Market | Settle  | 3600 DAI | 0      | 3700 DAI     | 1000         |
+
+The Vault is now holding 3700 DAI in total assets and 1000 shares. If Alice redeems her shares, she will receive 370 DAI.
```

Comment:
Alice has already redeemed her shares, back in Step 4, hasn't she?

Reply:
She can't redeem. She has no shares.

Reply:
Fair.